torch_ort/_version.py

```diff
@@ -0,0 +1 @@
+__version__ = '1.9.0.dev20220419'
```

Comment:
Should this file be part of the PR?

Reply:
No. It's removed. The file was added building torch_ort locally. Thank you for catching this!